src/Provider/TaxRateProvider.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Provider;
+
+use Doctrine\Common\Collections\Collection;
+use Sylius\Component\Core\Model\AdjustmentInterface;
+use Sylius\Component\Core\Model\OrderItemUnitInterface;
+
+final class TaxRateProvider implements TaxRateProviderInterface
+{
+    public function provide(OrderItemUnitInterface $orderItemUnit): ?string
```

Comment:
I wonder, should we pass there `$orderItemUnit` or maybe tax adjustment would be more suitable? The order item unit may or may not contain tax adjustment (therefore, we're forced to allow a null return). I feel like this check should be done before and not even call this service if we have no tax adjustment to process 🤔

Reply:
I guess that the goal is to get a tax rate for a refunded order item unit. In the implementation of this service, there is an assumption that we take the first adjustment, but this may not be the best solution for all cases and for me, this kind of service should decide about that. So, passing here `$orderItemUnit` seems to me a better option to achieve the original goal.

Reply:
Makes sense 👍